src/perf_event/sampling/config/extra_record.rs

```diff
@@ -77,6 +77,7 @@ impl ExtraRecord {
 
     pub(crate) fn enable_in_raw_attr(&self, raw_attr: &mut RawAttr) {
         #[rustfmt::skip]
+        #[allow(clippy::let_unit_value)]
```

Comment:
Add a comment here to explain why.

Reply:
This line has now been removed as there is no need to bind the match expression to `val`, see new push.

Reply:
Awesome, merged.